tests/components/ScriptSetupWithBuiltinComponent.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <component :is="Hello" />
```

Comment:
shouldn't that use `computedProperty`?

Reply:
@cexbrayat This is what I meant when I added the note. It does not matter whether you _use_ the computed property, its name is still what `shallowMount` will give to `component`. That's how weird this bug is.

Reply:
I'll add a third test to better outline the issue.